dblookupext/export_test.go

```diff
@@ -0,0 +1,13 @@
+package dblookupext
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+)
+
+func (hr *historyRepository) computeMiniblockHash(miniblock *block.MiniBlock) []byte {
```

Comment:
this could stay in a test file, since we don't export anything here

Reply:
refactored